apis/infix/atrium-api-infix/src/commonTest/kotlin/ch/tutteli/atrium/api/infix/en_GB/CharSequenceExpectationsTest.kt

```diff
@@ -41,5 +45,11 @@ class CharSequenceExpectationsTest : AbstractCharSequenceExpectationsTest(
 
         a1 = a1 toMatch Regex(".+Robert")
         a1 = a1 notToMatch Regex("a")
+
+        a1 = a1 length { it toEqual 23 }
+        a2 = a2 length { it toEqual 0 }
+
+        val l1: Expect<String> = a1 length {}
+        val l2: Expect<String> = a2 length {}
```

Comment:
that's the same overloads as in line 49,50, you can see that it is wrong as we expect that l1 and l2 are `Expect<Int>` (compare this with fluent)